### ISSUE TITLE
Fan going crazy on ASUS GL553VE

### ISSUE THREAD

**Issue by cosmincrihan** [marcoDallas/acpi_call_GUI_systemd#3]
I tried this method for disabling my discrete GPU (GTX 1050 Ti mobile), but the result is the same as with bbswitch: after disabling the card, the fan goes nuts and stays like that until I entirely power off the laptop.

The difference is that acpi_call_GUI does not freeze the system.
The ACPI call that it finds is: **\_SB.PCI0.PEG0.PEGP._OFF**

I am using the latest kernel, 5.8.3.

**Comment by marcoDallas** [comment]
Hi, situations like yours can happen, they depend a lot on the system configuration.
With acpi_call the only thing you can do is to try other call codes.

By the way, hybrid graphics should be supported with recent kernels like yours, have you tried some different Linux distributions? Eventually, you can check your BIOS and see if you can disable the discrete graphics card.

**Comment by cosmincrihan** [comment]
I tried everything I could and unfortunately, no luck whatsoever, even with other call codes.
Speaking of other distros, this is not an option for me because I have been using Debian since forever, and I found it very stable and bulletproof.

My BIOS does not offer any working options to disable the discrete card, and I searched for any mods that are available. There are mods for it that unlock the graphics options, but if you meddle with them, you might end with no screen at all.

The solution for me came up while searching for the ACPI bug for ASUS laptops, finally:
[https://bugzilla.kernel.org/show_bug.cgi?id=156341#c145](url)

It seems that PM for newer laptops is already present and all I need to do is:

```
echo 'auto' > '/sys/bus/pci/devices/0000:01:00.0/power/control'
rmmod nvidia
```

Also, uninstall `nvidia-persistenced` if present, so that the `nvidia` module does not get reloaded once in a while.

And now, powertop reports approximately -15W power consumption and I am happy as hell.
